libc/include/llvm-libc-macros/limits-macros.h

```diff
@@ -15,13 +15,16 @@
 #ifdef __clang__
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wgnu-include-next"
-#elif defined(__GNUC__) // gcc
-#if !defined _GCC_LIMITS_H_
-#define _GCC_LIMITS_H_
-#endif
+#else // gcc
 #pragma GCC system_header
 #endif // __clang__, __GNUC__
 
+#if defined(__GNUC__) && !defined(_GCC_LIMITS_H_)
```

Comment:
careful, clang's preprocessor also defines `__GNUC__` "for compatibility."

We only need to #define this for GCC due to GCC's limits.h (specific to GCC's resource dir, which is unused when CC=clang).

Reply:
Updated.